src/DBus.vala

```diff
@@ -56,6 +58,23 @@ public class Notifications.Server : Object {
 
         settings = new GLib.Settings ("io.elementary.notifications");
         bubbles = new Gee.HashMap<uint32, Notifications.Bubble> ();
+        valid_apps = new List<string> ();
```

Comment:
We create an app_info for the notification anyways, so doing this ahead of time is actually more expensive than just checking it as notifications come in

Reply:
I had a feeling that this could have been done in a nicer way, but I couldn't think about how at the time. :ok_hand: